#internal parameters for defining experimental parameters relevant to the gui
experiment_params:
  robot_names: ['Ground1', 'Aerial1']
  robot_commands: ['Pause', 'Resume', 'Soft e-stop', 'Hard e-stop', 'Define waypoint', 'Return home', 'Drop comms', "Highlight robot"]

#parameters for communicating with the DARPA command post
darpa_params:
  auth_bearer_token: ['flux230{showroom']
  scoring_uris: ['http://api/status', 'http://api/artifact_reports']
  artifact_categories: ['survivor', 'fire extinguisher', 'phone', 'backpack', 'drill']
  artifact_labels: [4, 3, 5, 1, 2]


